src/sas/qtgui/Perspectives/Corfunc/CorfuncPerspective.py

```diff
@@ -6,6 +6,7 @@
 # global
 from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg \
     as FigureCanvas
+from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT
```

Comment:
We don't use the Qt toolbar elsewhere, instead relying on our own Plotter class with a number of context menu items.

Reply:
This might not be a serious issue, considering the relevant charts are added as indices to the dataset in the Data Explorer and can be viewed with "Create New" or "Quick Plot"